src/components/plan/PlanModelErrorBar.svelte

```diff
@@ -0,0 +1,54 @@
+<svelte:options immutable={true} />
+
+<script lang="ts">
+  import { createEventDispatcher } from 'svelte';
+
+  export let hasErrors: boolean = false;
+
+  let isVisible: boolean = hasErrors;
+
+  $: isVisible = hasErrors;
+
+  const dispatch = createEventDispatcher<{
+    viewModelErrors: void;
+  }>();
+
+  function onClickViewConsole() {
+    dispatch('viewModelErrors');
+  }
+
+  function onDismiss() {
+    isVisible = false;
+  }
+</script>
+
+{#if isVisible}
+  <div class="model-error-bar">
+    <div class="info"></div>
```

Comment:
Could we have some sort of label here? Not sure what this bar is otherwise
<img width="1297" alt="image" src="https://github.com/NASA-AMMOS/aerie-ui/assets/4419607/015c3a07-60ab-4f7a-98b9-18070be21d70">